Week-2/assignment-1/index.html

```diff
@@ -0,0 +1,132 @@
+                                <h5>Lorem ipsum dolor sit amet consectetur adipisicing elit.</h5>
+                                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero deleniti iusto, minus
+                                    omnis delectus, accusantium, soluta sunt corporis ipsa est laudantium ipsum iure quia
+                                    perspiciatis? Repellendus mollitia quibusdam reprehenderit ut.</p>
+                                <a href="www.google.com">Link</a>
+                            </article>
+                            <article>
+                                <h5>Lorem ipsum dolor sit amet consectetur adipisicing elit.</h5>
+                                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero deleniti iusto, minus
+                                    omnis delectus, accusantium, soluta sunt corporis ipsa est laudantium ipsum iure quia
+                                    perspiciatis? Repellendus mollitia quibusdam reprehenderit ut.</p>
+                                    <a href="www.google.com">Link</a>
+                            </article>
+                            <article>
+                                <h5>Lorem ipsum dolor sit amet consectetur adipisicing elit.</h5>
+                                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero deleniti iusto, minus
+                                    omnis delectus, accusantium, soluta sunt corporis ipsa est laudantium ipsum iure quia
+                                    perspiciatis? Repellendus mollitia quibusdam reprehenderit ut.</p>
+                                    <a href="www.google.com">Link</a>
+                            </article>
+                        </section>
+                    </div>
+            </section>
+
+
+            <section class="form-container">
+                <form action="" onsubmit="">
+                    <label for="title">Title
+
+                        <span>
+                            Mr <input type="radio" name="title" value="Mr">
+                            Mrs <input type="radio" name="title" value="Mrs">
+                            Miss <input type="radio" name="title" value="Miss">
+                        </span>
+
+                    </label>
+
+                    <label for="firstname">
+                        First Name <input type="text" id="firstname" name="firstname" required="true">
+                    </label>
+                    <label for="lastname">
+                        Last Name <input type="text" id="lastname" name="lastname" required="true">
+                    </label>
+
+                    <label for="email">
+                        Email <input type="email" id="email" name="email" required="true">
+                    </label>
+
+                    <label for="phonenumber">
+                        Phone Number <input type="text" id="phonenumber" name="phonenumber" required="true">
```

Comment:
type can we tel. And could add a min length check as well

Reply:
added